From: Rob Clark <rob@ti.com>
Date: Sun, Feb 27, 2011 at 9:07 PM
Subject: [pandaboard] [PATCH] don't spam log with dump_stack()!!
To: pandaboard@googlegroups.com
Cc: Rob Clark <rob@ti.com>

Someone left some debug code, at least in the 2.6.35 ubuntu kernel..  It
isn't an error, and if you want to stop getting your logs spammed you
can use this patch.

diff -Naur linux-2.6.35-980.1release14/arch/arm/plat-omap/omap_device.c linux-2.6.35-980.1release14a/arch/arm/plat-omap/omap_device.c
--- linux-2.6.35-980.1release14/arch/arm/plat-omap/omap_device.c	2011-01-19 11:27:25.000000000 +0100
+++ linux-2.6.35-980.1release14a/arch/arm/plat-omap/omap_device.c	2011-02-28 16:22:25.689516696 +0100
@@ -890,7 +890,6 @@
 	if (!strcmp(dev_name(dev), "l3_main_1.0")) {
 		pr_debug("In--> %s,reqdev %s, dev %s to %ld rate\n",
 			__func__, dev_name(req_dev), dev_name(dev), rate);
-		dump_stack();
 	}
 
 	/*
